blockbase/theme.json

```diff
@@ -239,6 +239,7 @@
 			},
 			"gap": {
 				"baseline": "15px",
+				"header": "80px",
```

Comment:
Maybe it's a nitpick, but I feel like it's hard to guess here that we are only talking about the bottom padding without looking at the template part. Maybe this should be something like:

```
"header": {
    "bottom": "80px",
    "top": "var(--wp--custom--gap--vertical)"
}
```

and remove [this line from the CSS](https://github.com/Automattic/themes/blob/029c1c316f348d02cfb18356b24632b6f242479c/blockbase/sass/base/_header.scss#L4). It feels closer to what GB may do in the future and it's customizable by the user from the FSE.